README.md

```diff
@@ -1,12 +1,27 @@
-# Git Repository Template
+# CryptoPrices iOS
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+This is a simple iOS application built with SwiftUI that shows the crypto market prices.
 
-## Usage
+## Build and run
+Open your terminal, then:
 
-Clone the repository
+1. Checkout source code
+   ```
+   git clone git@github.com:nimblehq/nimble-crypto-ios.git
+   cd nimble-crypto-ios
+   ```
 
-`git clone git@github.com:nimblehq/git-template.git`
+2. Install dependencies
+   ```
+   make setup
+   ```
+
+3. Open `CryptoPrices.xcodeproj` with Xcode, hit `⌘R` to run.
+
+4. When adding/removing/updating any resources (colors and images assets, fonts, localized strings), please run the following before re-running the app from Xcode:
+   ```
+   make generate
+   ```
```

Comment:
@minhnimble Yes, I feel that too 🥲

Usually, when we put the resources in the app, we can just add a script in Xcode's build phase so that it can run every time we build (or run) by Xcode. In our case, however, we distribute resources in multiple packages. Of-course we can still do that but it's very strange that to generate code for a module we don't build that module itself but the whole app altogether.

Fortunately, there're some ways 😄:
- Run `swiftgen` every time a folder is changed. This is [documented](https://github.com/SwiftGen/SwiftGen/blob/stable/Documentation/Articles/Watch-a-folder-for-changes.md) by `swiftgen`. This is nice but introduces another dependency ([kicker](https://github.com/alloy/kicker) if we follow their recommendation. It's a Ruby gem, and runs like a daemon server in the background)
- At WWDC 2022, Apple introduced [Swift Package plugins](https://developer.apple.com/videos/play/wwdc2022/110359/) which allow developers to perform actions on Swift packages (which include pre-build action). It's available from Xcode 14 and Swift 5.6. This is exactly what we need and should go for. We can write our own plugin or use one written by someone else [here](https://github.com/SwiftGen/SwiftGenPlugin).

But I think this deserves a dedicated task since we'll have to explore more on Apple documentation and decide to write our own or use a third-party plugin. What do you think 🙏

Reply:
@Thieurom nice investigation 👍 I thought we can just simply add a script in Xcode's build phase but yeah you made valid points there! Let's work on a separate task to handle [Swift Package plugins](https://developer.apple.com/videos/play/wwdc2022/110359/) then. 🚀